src/main/java/com/cloudbees/jenkins/support/impl/AboutJenkins.java

```diff
@@ -500,7 +494,7 @@ private static String humanReadableSize(long size) {
                 out.println("  * Mode:    WAR");
             }
             final JenkinsLocationConfiguration jlc = JenkinsLocationConfiguration.get();
-            out.println("  * Url:     " + (jlc != null ? jlc.getUrl() : "No JenkinsLocationConfiguration available"));
+            out.println("  * Url:     " + (jlc != null ? (filter == null ? jlc.getUrl() : filter.filter(jlc.getUrl())) : "No JenkinsLocationConfiguration available"));
```

Comment:
This would be the other way actually?

Reply:
:thinking: What do you mean? If the filter is null we return the url without filtering. If the filter is not null, we filter the url. The code seems fine to me. Do you mean do ```filter != null ? filter.filter(...```?

Reply:
Yes, my mistake sorry. Anyway could jlc.getUrl() be null and, in that case, does the filter handle that correctly?

Reply:
I have changed this: https://github.com/jenkinsci/support-core-plugin/pull/174/commits/753c58cb285ada8ccd4efa4743345fcdb8056533#diff-f2c7ab60306f05af2fab348ce75750cfR498